blog/enforce-cloud-security-compliance/index.mdx

```diff
@@ -0,0 +1,142 @@
+---
+date: 2023-06-06
+authors: [matthias]
+tags: [security, aws, resoto]
+---
+
+# Enforce Cloud Security Compliance with Resoto
+
+In our rapidly evolving digital landscape, cybersecurity has become a non-negotiable aspect of doing business. Now more than ever, organizations are recognizing the criticality of implementing stringent security compliance in their cloud infrastructure. A security baseline, as we discussed in our [previous blog post](../cloud-resource-security-benchmarks/index.mdx), forms the foundation of this effort.
```

Comment:
Replace This  (previous blog post ) with (Cloud Resource Security Benchmarks) for Topic Related Anchor text.